portfolio/src/main/webapp/script.js

```diff
@@ -74,42 +79,47 @@ function createListElement(text) {
 }
 
 /** Deletes all comments. */
-function deleteAllComments() {
+function deleteAllComments(blogNumber) {
   console.log("Deleting all comments");
-  fetch('/delete-comment', {method: 'POST'}).then(loadCommentsSection);
+  var queryString = '/delete-comment?blog-number=' + blogNumber;
+  fetch(queryString, {method: 'POST'}).then(() => {
+    loadCommentsSection(blogNumber);
+    });
 }
 
 /** Creates pagination to go through all comments. */
-function loadPagination(numComments) {
+function loadPagination(numComments, blogNumber) {
   var queryString = '/pagination-comment?num-comments=' + numComments;
+  queryString = queryString + '&blog-number=' + blogNumber;
 
-  console.log("Fetching comments");
+  console.log("Fetching pagination for blog post " + blogNumber);
   fetch(queryString).then(response => response.json()).then((maxPageNum) => {
-    const paginationElement = document.getElementById('pagination');
-    console.log("Loading pagination");
+    const paginationElement = document.getElementById('pagination-' + blogNumber);
+    console.log("Loading pagination w/ " + numComments + " comments per page");
     paginationElement.innerHTML = '';
       for (var i = 1; i < maxPageNum + 1; i++) {
         paginationElement.appendChild(
-          createPageElement(i, numComments));
+          createPageElement(i, numComments, blogNumber));
       }
   });
 }
 
 /** Creates an elemet that represents a page */
-function createPageElement(pageNumber, numComments) {
-  console.log("Creating page number " + pageNumber);
+function createPageElement(pageNumber, numComments, blogNumber) {
+  console.log("Creating page number " + pageNumber + " for blog post " + 
+  blogNumber);
```

Comment:
Continued lines should be double-indented (+4 spaces total)

```suggestion
      blogNumber);
```